### PR TITLE
Add html-css-popup

### PR DESCRIPTION
**Pull request by misshatsker**
### html-css-popup

[Demo](https://misshatsker.github.io/hooli-style-popup/) |
[Code base](https://github.com/misshatsker/hooli-style-popup)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

File: submissions/misshatsker/html-css-popup/index.css [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+    padding: 10px;
+    border: 1px solid transparent;
+}
+
+.profile-link {
+    display: block;
+    border: 1px solid transparent;
+}
+
+.link:hover, .link:focus, .profile-link:focus {
```
Comment (HannaSyn) [inline submissions/misshatsker/html-css-popup/index.css:73]: It is better to start every class name after coma from a new line

**Review by HannaSyn — CHANGES_REQUESTED** [review]
Cute work! See my comment below to improve it

**Comment by misshatsker** [comment]
@HannaSyn thanks a lot for your review! Could you please take a look at my changes/fixes?

**Comment by HannaSyn** [comment]
Looks great 🤩

**Pull request merged**